### PR TITLE
Added a modern variant in Experience section

### PR DESCRIPTION
**Pull request by Ankitagupta16**
Desktop View
firstPull
 
Mobile View
Mobile

### CODE REVIEW

File: src/modules/resume/experience/variants/modern/metadata.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+const metadata = {
+    name: "Modern",
+    description: "A modern variant",
```
Comment (AKAspanion) [inline src/modules/resume/experience/variants/modern/metadata.js:4]: @Ankitagupta16 Add more to this description.

File: src/modules/resume/experience/variants/modern/properties.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          },
+          location: {
+            name: 'Location',
+            type: propertyTypes.STRING,
+            value: 'Location',
+            required: true
+          },
+          period:{
+            name: 'Period',
+            type: propertyTypes.STRING,
```
Comment (AKAspanion) [inline src/modules/resume/experience/variants/modern/properties.js:53]: @Ankitagupta16 propertyTypes.DATE?

File: src/modules/resume/experience/variants/modern/properties.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    required: true
+  },
+  headlineText: {
+    name: 'Headline text',
+    type: propertyTypes.STRING,
+    value: 'Experience',
+    required: true
+  },
+  experience: {
+    name: 'Experience',
```
Comment (AKAspanion) [inline src/modules/resume/experience/variants/modern/properties.js:18]: name: 'Experience', repeated in value below.
missing the required key.
@Ankitagupta16

File: src/modules/resume/experience/variants/modern/properties.js (hunk 2 above)
Comment (Ankitagupta16) [reply src/modules/resume/experience/variants/modern/properties.js:53]: propertyTypes.DATE won't work for  this case : ( 2019- Present)
Thus I thought to make it STRING type.

File: src/modules/resume/experience/variants/modern/metadata.js (hunk 1 above)
Comment (Ankitagupta16) [reply src/modules/resume/experience/variants/modern/metadata.js]: Done

File: src/modules/resume/experience/variants/modern/properties.js (hunk 3 above)
Comment (Ankitagupta16) [reply src/modules/resume/experience/variants/modern/properties.js]: Done

File: src/modules/resume/experience/variants/modern/render.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    headlineText,
+    experience
+  }
+) => {
+  const styles = useStyles();
+
+  function renderExperience(experienceDetails) {
+    let description=[]
+    for(let i=0;i<4;i++){
+      description.push(<li><p style={styles.experienceItem}>{experienceDetails.description.value}</p></li>)
```
Comment (sahaparamjit) [inline src/modules/resume/experience/variants/modern/render.js:30]: Create a function to render the experience bullet points using map function.

Comment (Ankitagupta16) [reply src/modules/resume/experience/variants/modern/render.js]: Done, please review

**Pull request closed without merging**